network-api/networkapi/wagtailpages/pagemodels/taxonomy.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from wagtail.admin import FieldPanel
+
+
+class BaseTaxonomy(models.Model):
```

Comment:
It would be good to have the `TranslatableMixin` here, as I think all (most?) taxonomies are translatable.

Reply:
:+1:  Added in 4ec53609107b1a6924596cd4972bfb0ed38bf084